src/main/java/step3/domain/BasicForwardStrategy.java

```diff
@@ -0,0 +1,20 @@
+package step3.domain;
+
+import step3.config.PowerConfig;
+
+import java.util.Random;
+
+public class BasicForwardStrategy implements ForwardStrategy {
+
+    private static final Random PICKER = new Random();
+
+    @Override
+    public int calculateForwardDistance() {
+        int power = PICKER.nextInt(PowerConfig.MAX);
+        if(power >= PowerConfig.CONDITION) {
+            return 1;
```

Comment:
매직넘버는 상수로 관리하는것이 어떨까요 ? 
https://www.slipp.net/questions/356
```suggestion
            return MOVE;
```

Reply:
이동거리를 계산하는거라 숫자를 리턴하려 했는데 스펙이 1, 0으로 정해져서 상수로 하는게 좋을 것 같습니다!